### PR TITLE
docs: Added information for Sancho GovTool Beta

### PR DESCRIPTION
**Pull request by Ryun1**
### Changes
- Added page for Sancho GovTool Beta
- Added Sancho GovTool beta links to resource page
- Reordered the tutorials section to make more sense
 
#### New page
scrnli_22_11_2023_12-08-52

#### Links added 
image

#### Reorder
image

### CODE REVIEW

**Review by michaelmadoff — APPROVED** [review]

File: docs/get-started/govtool.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+sidebar_position: 3
+---
+
+# SanchoNet GovTool
+
+![SanchoNet GovTool Logo](../../static/img/sanchonet-govtool-header.png)
+
+## What is the SanchoNet GovTool?
+
+The SanchoNet GovTool allows ada holders to experience now some of the governance features described in [CIP-1694](https://www.1694.io/) and to test governance features in SanchoNet in a guided and simple experience.
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:12]:
```suggestion
The SanchoNet GovTool enables ada holders to experience some of the governance features described in [CIP-1694](https://www.1694.io/) and to test governance features in SanchoNet through a guided and straightforward experience. 
```

File: docs/get-started/govtool.mdx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+The SanchoNet GovTool is open for beta testing now and it's accessible from [sanchogov.tools](https://sanchogov.tools/)
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:13]:
```suggestion
The SanchoNet GovTool is currently open for beta testing and can be accessed at [sanchogov.tools](https://sanchogov.tools/).
```

File: docs/get-started/govtool.mdx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## What is the Voltaire GovTool?
+
+The Voltaire GovTool, of which the SanchoNet GovTool is the testnet version, will enable ada holders to register to become a Delegated Representative (DRep), delegate voting power to DReps, and review and vote on governance actions.
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:17]:
```suggestion
The Voltaire GovTool, with the SanchoNet GovTool serving as its testnet version, will empower ada holders to register as delegate representatives (DReps), delegate voting power to DReps, and review and vote on governance actions. 
```

File: docs/get-started/govtool.mdx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Governance tooling is owned and maintained by the community.
+With community approval, Intersect, through its committees and working groups, will play an important role in hosting and maintaining these governance tools.
+Through active community involvement, the aim is to create the initial foundation layer together with different builders in the community.
+These governance tools will support the whole process, including both off-chain and on-chain, and as a whole they will support the implementation of the governance described in CIP-1694 and its adoption.
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:21]:
```suggestion
These governance tools will support the entire process, encompassing both off-chain and on-chain aspects. As a whole, they will facilitate the implementation of the governance described in CIP-1694 and its adoption.
```

File: docs/get-started/govtool.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+Once this foundation layer is ready, the community will be able to further contribute to these governance tools by maintaining and improving existing features and pillars, creating new features or completely new areas either by themselves or supported by awards and grants facilitated by Intersect.
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:22]:
```suggestion
Once this foundational layer is in place, the community will have the opportunity to enhance these governance tools. They will be able to achieve this by maintaining and improving existing features and pillars, creating new features, or establishing entirely new areas either independently or with support from awards and grants facilitated by Intersect.
```

File: docs/get-started/govtool.mdx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+With the freedom to test and refine Cardano’s next era of governance principles under one central apparatus, the GovTool helps the Cardano community forge a future governance framework that represents the community’s best democratic interests.
+
+## How to participate in the Beta Testing?
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:25]:
```suggestion
## How to participate in the beta testing?
```

File: docs/get-started/govtool.mdx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+The Voltaire GovTool will be one of the first tools that allows ADA holders to experience the CIP-1694 Governance on Cardano.
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:27]:
```suggestion
The Voltaire GovTool will be one of the first tools to allow ada holders to experience the CIP-1694 governance on Cardano.
```

File: docs/get-started/govtool.mdx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+The current version of this tool has been developed using initial feedback from the community and following what's described in CIP-1694.
+It's now important that the community tests this tool and provide valuable insights to further refine it and make it ready for future product use.
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:29]:
```suggestion
It's now important that the community tests this tool and provides valuable insights to further refine it and make it ready for future product use.
```

File: docs/get-started/govtool.mdx [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+Check the [beta testing to-do list](https://docs.sanchogov.tools/about/sanchonet-govtool-testing)
+Check the [prerequisites to start](https://docs.sanchogov.tools/how-to-use-the-govtool/prerequsites)
+Check the [quick/start guide](https://docs.sanchogov.tools/how-to-use-the-govtool/overview-quick-start)
+
+## Related guides and resources
+
+[SanchoNet Govtool guides](https://docs.sanchogov.tools/)
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:37]:
```suggestion
[SanchoNet GovTool guides](https://docs.sanchogov.tools/)
```

File: docs/get-started/govtool.mdx [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+Overall Governance Guides
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:38]:
```suggestion
General governance guides
```

File: docs/get-started/govtool.mdx [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+Cardano Governance Page overview
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:39]:
```suggestion
Cardano governance page overview
```

File: docs/get-started/govtool.mdx [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+Govtool overview
```
Comment (olgahryniuk) [inline docs/get-started/govtool.mdx:40]:
```suggestion
GovTool overview
```

File: docs/tools-resources/resources.mdx [hunk 13]
```diff
@@ -4,7 +4,14 @@ title: 'Tools and resources'
 sidebar_position: 1
 ---
 
-*Last update: 2023-11-02*
+*Last update: 2023-11-22*
+
+### SanchoNet GovTool Beta
```
Comment (olgahryniuk) [inline docs/tools-resources/resources.mdx:9]:
```suggestion
### SanchoNet GovTool beta
```

File: docs/tools-resources/resources.mdx [hunk 14]
```diff
@@ -4,7 +4,14 @@ title: 'Tools and resources'
...
+
+* [SanchoNet GovTool Beta Site](https://sanchogov.tools)
```
Comment (olgahryniuk) [inline docs/tools-resources/resources.mdx:11]:
```suggestion
* [SanchoNet GovTool beta site](https://sanchogov.tools)
```

File: docs/tools-resources/resources.mdx [hunk 15]
```diff
@@ -4,7 +4,14 @@ title: 'Tools and resources'
...
+* [Documentation Site](https://docs.sanchogov.tools/)
```
Comment (olgahryniuk) [inline docs/tools-resources/resources.mdx:12]:
```suggestion
* [Documentation site](https://docs.sanchogov.tools/)
```

**Comment by Ryun1** [comment]
@disassembler @CarlosLopezDeLara 😎

**Review by CarlosLopezDeLara — APPROVED** [review]

**Review by disassembler — APPROVED** [review]

**Review by disassembler — APPROVED** [review]

**Pull request merged**